DQM/DTMonitorModule/src/DTLocalTriggerLutTask.cc

```diff
@@ -265,12 +300,44 @@ void DTLocalTriggerLutTask::searchDccBest( std::vector<L1MuDTChambPhDigi> const*
     int st   = trigIt->stNum();
     int qual = trigIt->code();
 
-    if(qual>trigQualBest[wh+3][st][sec] && qual<7) {
-      trigQualBest[wh+3][st][sec]=qual;
-      trigBest[wh+3][st][sec] = &(*trigIt);
+    if(qual>trigQualBestIn[wh+3][st][sec] && qual<7) {
```

Comment:
Yes, but these pieces of code are not really mine, I have just followed the coding trend used by the creator of the class...
On the other hand, I don't understand what you mean: wh is not constant, and something like:
    int wh   = trigIt->whNum();
    const int whIndex = wh+3;
does not compile.... what do you exactly mean?